tests/abi/tools/revng-ensure-rft-equivalence/Main.cpp

```diff
@@ -177,6 +177,28 @@ int main(int Argc, char *Argv[]) {
     Iterator->second.Right = Right->key();
   }
 
+  // Deduplicate the list of IDs to replace.
```

Comment:
Why do we need to do this?

Reply:
Because after importing PDBs we end up with the same prototype (function type) used for multiple functions. And since this is gathered function-wise, we end up with weirdness because it tries to replace the same multiple times.

Reply:
Comment

Reply:
Ping?

Reply:
Done:
```cpp
  // We have to do this because in case a single prototype is used for multiple
  // functions (common in binaries lifted from PE/COFF) we still only want to
  // "replace" it once.
```